docs/src/componentDocs/Header/markdown/HeaderAPIDocs.mdx

```diff
@@ -1 +1,135 @@
+| backgroundImage     | An image to blend with the colored background in the header                                                                | `ImageSourcePropType`                                                            | no       |                                   |
+| collapsedHeight     | The height of the header when collapsed                                                                                    | `number`                                                                         | no       | 56                                |
+| expandable          | Allow the header to expand/collapse on tap                                                                                 | `boolean`                                                                        | no       | `false`                           |
+| expandedHeight      | The height of the header when expanded                                                                                     | `number`                                                                         | no       | 200                               |
+| fontColor           | Color of the title, subtitle, info, and icons in the header                                                                | `string`                                                                         | no       | `theme.colors.onPrimaryContainer` |
+| info                | Third line of text (hidden on collapse)                                                                                    | `ReactNode`                                                                      | no       |                                   |
+| icon                | Icon to show to the left of the title                                                                                      | [`IconSource`](/components/Icons/api-docs)                                                       | no       |                                   |
+| navigationIconColor | The color used for the Navigation icon                                                                                     | `string`                                                                         | no       | `theme.colors.onSurface`          |
+| onIconPress         | A callback to execute when the icon is pressed                                                                             | `() => void`                                                                     | no       |                                   |
+| scrollPosition      | Y-value of the linked ScrollView (dynamic variant only)                                                                    | `Animated.Value`                                                                 | no       |                                   |
```

Comment:
scrollPosition seems to have default https://github.com/etn-ccis/blui-react-native-component-library/blob/6c996dc02c884f60cfdc62938e93836e6d641666/components/src/core/Header/Header.tsx#L244